phoenix-core/src/it/java/org/apache/phoenix/end2end/IndexScrutinyToolBaseIT.java

```diff
@@ -54,7 +55,7 @@
     }
 
     protected List<Job> runScrutiny(String[] cmdArgs) throws Exception {
-        IndexScrutinyTool scrutiny = new IndexScrutinyTool();
+        IndexScrutinyTool scrutiny = new IndexScrutinyTool(IndexScrutinyMapperForTest.class);
```

Comment:
I would suggest either of these approached 1. Just override hasRowExpiredOnSource() in IndexScrutinyMapperForTest class and do IndexScrutinyTool scrutiny = new IndexScrutinyMapperForTest(), OR 2. Use Callable<> to pass in a function implementation. I like the former approach.

Reply:
I made changes to override only a small part of the function. Please review.